cartography/intel/gcp/__init__.py

```diff
@@ -27,4 +115,12 @@ def start_gcp_ingestion(session, config):
             e
         )
         return
-    crm.sync_gcp_organizations(session, credentials, config.update_tag, common_job_parameters)
+    resources = _initialize_resources(credentials)
+
+    # If we don't have perms to pull Orgs or Folders from GCP, we will skip safely
+    crm.sync_gcp_organizations(session, resources.crm_v1, config.update_tag, common_job_parameters)
+    crm.sync_gcp_folders(session, resources.crm_v2, config.update_tag, common_job_parameters)
+
+    projects = crm.get_gcp_projects(resources.crm_v1)
+
```

Comment:
Please add a comment reflecting that we currently don't handle individual projects outside of an organization. We could also add an issue for this -- it may be a good first issue for someone to work on.

Reply:
Added code handling projects without a parent.